### ISSUE TITLE
[Due for payment 2026-04-01] Screen Reader: Many Pages: The user is not informed about displayed time and when it disappears

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77259]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Using iOS VoiceOver or Android TalkBack, open the Expensify app
2. Swipe to the 'Phone and email' field
3. Type the valid email address and double-tap to activate
4. Note that the time counter appeared on the screen
5. Note what is announced

Other Occurrences:
1. Onboarding - Verify your email
2. On Profile - Verify Email
3. On Security - Merge Accounts
4. On Security - Merge Accounts - Enter Code

## Expected Result:
The displayed time should be announced once it appears, e.g.:
- 'Time remaining: 19 seconds'
And immediately announce the final status when the timer ends: 'The time has expired'

## Actual Result:
The time display changes visually, but VoiceOver remains silent, leaving the user unaware of how much time has passed or when the time limit has passed.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 4.1.3 Status Messages
https://www.w3.org/WAI/WCAG22/Understanding/status-messages

## User impact:
When appropriate roles or properties are assigned to status messages, the new content is spoken by screen readers in such a way as to assist blind and low vision users. Assigning proper roles or properties to status messages provides possible future uses and personalization opportunities, such as the potential to be exploited by assistive technologies created for users with some cognitive disabilities.

## Suggested resolution:
Ensure the displayed time is announced once it appears, e.g.:
- 'Time remaining: 19 seconds'
And immediately announce the final status when the timer ends: 'The time has expired'
https://appt.org/en/docs/react-native/samples/accessibility-live-region

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

<img width="828" height="1792" alt="Image" src="https://github.com/user-attachments/assets/f771089f-09da-4923-965d-073800f25eac" />

https://github.com/user-attachments/assets/05b2b374-383a-48ce-8c2e-19ee4feaa8f7

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005413
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by laurenreidexpensify** [comment]
Payment Approved - $250 in New Expensify @situchan
